rmw_connextdds_common/include/rmw_connextdds/context.hpp

```diff
@@ -91,6 +92,23 @@ struct rmw_context_impl_t
   std::string qos_ctx_namespace;
   bool override_publish_mode;
 
+  enum class qos_profile_loading_policy_t
+  {
+    // Use default QoS policy got from the DDS qos profile file applying topic filters
+    // and apply the ROS settings on top of it.
+    Always,
+    // Use default QoS policy got from the DDS qos profile file applying topic filters
+    // without any changes for the topics that matches the provided regex expressions.
+    // Apply the ROS settings on top of it for the ones that doesn't match.
+    DDSTopics,
+    // Use default QoS policy got from the DDS qos profile file applying topic filters
+    // without any changes.
+    Never,
+  };
+
+  qos_profile_loading_policy_t qos_profile_loading_policy;
+  std::regex dds_topics_regex;
```

Comment:
I would qualify this member a bit further, e.g. `std::regex endpoint_qos_override_policy_topics_regex`, just to "explicitate" more its relationship with the other field.

Reply:
Done https://github.com/rticommunity/rmw_connextdds/pull/7/commits/7e5fbe48e853c4708acbcf2e2d36f88f1f0f8cc9